include/seastar/http/httpd.hh

```diff
@@ -210,6 +213,25 @@ private:
     friend class http_server_tester;
 };
 
+template<typename Clock = std::chrono::steady_clock>
+future<> http_server::graceful_pre_stop(int timeout_ms) {
+    future<> f = _task_gate.close();
+    shared_future<with_clock<Clock>> shared_done(std::move(f));
+    _tasks_done = shared_done.get_future();
+    for (auto &&l : _listeners) {
```

Comment:
Code style nitpick: in Seastar, && sticks to the type - `auto&& l`. The same comment applies to all other type declaration in this series

Reply:
fixed